Updates to non-functional requirements (#26)
nordic-institute/X-Road-development: NFR.md

5.1.4 Source Code MUST be marked with licensing information according to
MIT license rules. Copyright MUST be clearly defined.

> 5.1.5 New X-Road components MUST be created using the same Java version with the X-Road core.

Reviewer: Perhaps using the term `X-Road core` is a bit unclear for readers outside the project in this context, since there isn't one specific module we can direct the users at for guidance. Maybe it would be better to say it like this:
```suggestion
5.1.5 New X-Road components MUST be created using the same Java version as the existing components.
```

7.1.2 The component is allowed to send e-mails only through an
external mail agent (for example, postfix).

> 7.1.3 All distributed components of the application MUST be packaged for the 
> platforms defined in section 3.5. Source code MUST be supplemented with 
> scripts required to build packages for the supported platforms.

Reviewer: This place is a bit tricky, since for Docker we decide what is included in the containers, so we couldn't say that all distributed packages are necessarily packaged for Docker. E.g. currently the HWtoken addon isn't.

So maybe it would be good to say here something in the lines of:
```suggestion
7.1.3 All distributed components of the application MUST have native packaged for the 
platforms defined in section 3.5. Source code MUST be supplemented with 
scripts required to build native packages for the supported platforms. Docker instead
is a special case and packages included in the containers are determined by the maintainers
on a case-by-case basis.
```

7.3.4 Logs are written in English.

> 7.3.5 All web servers MUST produce `access.log` files in a standard

Reviewer: Maybe specifying a log file name here is bad, since I don't believe we produce this file specifically? So could we here instead say:
```suggestion
7.3.5 All web servers MUST produce files containing their access logs in a standard
```
